Formula/aria2.rb

```diff
@@ -37,9 +37,42 @@ def install
     system "./configure", *args
     system "make", "install"
 
+    (buildpath/"aria2.conf").write <<~EOS
+      enable-rpc=true
+      rpc-secret=barfoo
```

Comment:
Hardcoding a secret in all installs sounds bad.

Reply:
How about add a ```caveats``` block to inform users to create a configuration by themselves?